sklearn/linear_model/_least_angle.py

```diff
@@ -109,26 +109,26 @@ def lars_path(
         Controls output verbosity.
 
     return_path : bool, default=True
-        If ``return_path==True`` returns the entire path, else returns only the
-        last point of the path.
+        If ``True``, the entire path is returned. If ``False``, only the last point
```

Comment:
The previous docstring was fine. I would keep it as-is.

Reply:
Happy to revert, of course, just to check my understanding: I thought it was weird to see a comparison to `True` with `==`, which is generally discouraged.

Reply:
I've reverted the docstring to the previous version and changed the first part to follow the same pattern as the other boolean args.